code/modules/reagents/Chemistry-Recipes.dm

```diff
@@ -523,20 +525,75 @@
 	required_reagents = list("mindbreaker" = 1, "hydrazine" = 1)
 	result_amount = 3
 
-/datum/chemical_reaction/citalopram
-	name = "Citalopram"
-	id = "citalopram"
-	result = "citalopram"
+/datum/chemical_reaction/escitalopram
+	name = "Escitalopram"
+	id = "escitalopram"
+	result = "escitalopram"
 	required_reagents = list("mindbreaker" = 1, "carbon" = 1)
 	result_amount = 3
 
-
 /datum/chemical_reaction/paroxetine
 	name = "Paroxetine"
 	id = "paroxetine"
 	result = "paroxetine"
 	required_reagents = list("mindbreaker" = 1, "acetone" = 1, "inaprovaline" = 1)
 	result_amount = 3
+	
+/datum/chemical_reaction/fluvoxamine
+	name = "Fluvoxamine"
+	id = "fluvoxamine"
+	result = "fluvoxamine"
+	required_reagents = list("mindbreaker" = 1, "iron" = 1, "potassium" = 1)
+	result_amount = 3
+	
+/datum/chemical_reaction/sertraline
+	name = "Sertraline"
+	id = "sertraline"
+	result = "sertraline"
+	required_reagents = list("mindbreaker" = 1, "aluminum" = 1, "potassium" = 1)
+	result_amount = 3
+	
+/datum/chemical_reaction/paroxetine
+	name = "Paroxetine"
+	id = "paroxetine"
+	result = "paroxetine"
+	required_reagents = list("mindbreaker" = 1, "ammonia" = 1, "copper" = 1)
+	result_amount = 3
+	
+/datum/chemical_reaction/duloxetine
+	name = "Duloxetine"
+	id = "duloxetine"
+	result = "duloxetine"
+	required_reagents = list("mindbreaker" = 1, "silicon" = 1, "ethanol" = 1)
+	result_amount = 3
+	
+/datum/chemical_reaction/venlafaxine
+	name = "Venlafaxine"
+	id = "venlafaxine"
+	result = "venlafaxine"
+	required_reagents = list("mindbreaker" = 1, "sodium" = 1, "tungsten" = 1)
+	result_amount = 3
+	
+/datum/chemical_reaction/risperidone
+	name = "Risperidone"
+	id = "risperidone"
+	result = "risperidone"
+	required_reagents = list("mindbreaker" = 1, "space_drugs" = 1, "ethanol" = 1)
+	result_amount = 3
+	
+/datum/chemical_reaction/olanzapine
+	name = "Olanzapine"
+	id = "olanzapine"
+	result = "olanzapine"
+	required_reagents = list("mindbreaker" = 1, "space_drugs" = 1, "silicon" = 1)
+	result_amount = 2
+	
+/datum/chemical_reaction/trisyndicotin
+	name = "Trisyndicotin"
+	id = "trisyndicotin"
+	result = "trisyndicotin"
+	required_reagents = list("hextrasenil" = 1, "space_drugs" = 1, "radium" = 1)

```

Comment:
I dont see the recipe for hextrasenil.
Also dont base hextrasenil or trisyndicotin on the other.
They should both use a different antidepreassant as base but with different "addon" chems.